k8s/docs/values-stage.yaml

```diff
@@ -1,13 +1,9 @@
-# Default values for airqo-docs.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
 replicaCount: 1
 
 image:
   repository: gcr.io/airqo-250220/airqo-stage-docs
   pullPolicy: Always
-  tag: latest
+  tag: stage-090ad16f-1667922055
```

Comment:
I might need more details about these tag names. I am somehow confused now. Why these tags? What is the convention? @123MwanjeMike

Reply:
Those are the image tag names on gcr @Baalmart 
![image](https://user-images.githubusercontent.com/41787587/201306236-61612b5e-ea92-4449-8294-6f22f1e93a1e.png)